### DISCUSSION TITLE
Strava exports all show as "workout" ???

### DISCUSSION THREAD

**Discussion by gregoryxxxxxx** [mytourbook/mytourbook#1291] [Q&A]
I've been ignoring this issue for a long while... but figured I should ask eventually. The "Tour Type Filter" functions with the Strava:Run and Strava:Ride sort of filters used to work but don't seem to work for me anymore. Is there something I need to do differently? Or is it just broken now? Is it working for others? 

Thx.

**Comment by FJBDev** [comment]
@gregoryxxxxxx 
You can see [here](https://github.com/mytourbook/mytourbook/blob/main/bundles/net.tourbook.export/src/net/tourbook/export/fit/FitSportMapper.java#L28) more details on how the FIT sport type is mapped from the tour type (MT uploads each tour to Strava as a FIT file). 
And you can see more details [here](https://github.com/mytourbook/mytourbook/issues/845#issuecomment-1515472101) where I discussed it in the FIT export feature.

My hunch is that since you have the prefix `Strava:XXXXX` that prevents from mapping to the proper sport type.

**Reply by gregoryxxxxxx** [reply]
Thank you. Pointing to that list of FIT titles helped a lot. I guess I was confused when I built the Strava:activity filters way back. But now I can get them working with that list. Thank you.

**Comment by mff73** [comment]
Hi,
i would put a comment here, as it is related :)
@FJBDev
The mapping between garmin sports/subsports and MTB tour types are "hard coded", isn't it ?
What do you think about a new feature of MTB, to enable each user to map its own tour types to garmin fit official sports/subsports ?
like for example : adding two extra fields in "USER".TOURTYPE table with SPORT and SUB_SPORT : 
original table
![image](https://github.com/mytourbook/mytourbook/assets/50132146/7ecc5146-743d-49a3-a121-df5532c59610)

excel mockup 😊
![image](https://github.com/mytourbook/mytourbook/assets/50132146/29d36b37-25f1-4cf6-8a6b-de16e9ab3d89)

**Comment by FJBDev** [comment]
> Hi, i would put a comment here, as it is related :) @FJBDev The mapping between garmin sports/subsports and MTB tour types are "hard coded", isn't it ? What do you think about a new feature of MTB, to enable each user to map its own tour types to garmin fit official sports/subsports ?

Yes, that could be a possibility if users need to be able to map each tour type to a FIT Sport/SubSport combination.

Feel free to create a detailed feature request
